### PR TITLE
Add models for login security related doc links

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [X] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/extensions/configs/models/documentation.ts [hunk 1]
```diff
@@ -462,6 +468,8 @@ export interface DocumentationLinksExtensionInterface {
          * Documentation links for account recovery elements.
          */
         accountRecovery: AccountRecoveryDocumentationLinksInterface;
+
```
Comment (dasuni-30) [inline apps/console/src/extensions/configs/models/documentation.ts:471]: Let's add a Doc comment to maintain consistency

Comment (pavinduLakshan) [reply apps/console/src/extensions/configs/models/documentation.ts]: Fixed

**Review by dasuni-30 — APPROVED** [review]

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/server-configurations/pages/connector-edit-page.tsx [hunk 2]
```diff
@@ -419,7 +429,7 @@ export const ConnectorEditPage: FunctionComponent<ConnectorEditPageInterface> =
                         >
                             When self registration is enabled, users can register via the
                             <strong>
-                                Create an account
+                                Register
```
Comment (brionmario) [inline apps/console/src/features/server-configurations/pages/connector-edit-page.tsx:432]: Hope this is a unified experience?

Comment (pavinduLakshan) [reply apps/console/src/features/server-configurations/pages/connector-edit-page.tsx:432]: Yes, this is unified UX.

Screenshot 2024-03-24 at 13 20 24

**Pull request merged**